CMakeLists.txt

```diff
@@ -1,93 +1,41 @@
-######### CMake Version #####################
-cmake_minimum_required(VERSION 2.8.11)
-#############################################
+project(eigen-cuda LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.14)
 
-######### Options ###########################
-option( CORE_USE_CUDA    "Use CUDA to speed up certain parts of the code."              ON )
-#############################################
+option(USE_CUDA "Whether to use CUDA" OFF)
 
-
-######### CUDA decisions ####################
-if (CORE_USE_CUDA)
-    MESSAGE( STATUS ">> --------------  USING CUDA --------------" )
-    set( CUDA_TOOLKIT_ROOT_DIR "/opt/cuda" )
-    if (APPLE OR UNIX)
-        set(CMAKE_C_COMPILER   /opt/cuda/bin/gcc)
-        set(CMAKE_CXX_COMPILER /opt/cuda/bin/g++)
-    elseif (WIN32)
-        ### By default we use VS
-        MESSAGE( STATUS ">> User compiler:             MSVC" )
-        MESSAGE( STATUS ">> Choosing a different compiler is not yet implemented for Windows" )
-    endif()
-endif()
-#############################################
-
-
-######### Info ##############################
-MESSAGE( STATUS ">> CMAKE_C_COMPILER:        " ${CMAKE_C_COMPILER} )
-MESSAGE( STATUS ">> CMAKE_CXX_COMPILER:      " ${CMAKE_CXX_COMPILER} )
-#############################################
-
-
-######### Project Name ######################
-project(eigencuda)
-SET( EXECUTABLE_NAME run )
-#############################################
-
-
-### Find includes in corresponding build directories
-set( CMAKE_INCLUDE_CURRENT_DIR ON )
-######### Have the binary placed into the source head
```

Comment:
If you remove this section, the build does not match the docs, which say
> To run an example, do
> ```shell
> ./main
> ```

(on msvc, but also if you implement my comments on the readme)

Reply:
oh okay, running Ubuntu here, not sure I can look into this

Reply:
why did you remove the section placing the binary into the root cmake dir?